doc/manage-duniverse.mld

```diff
@@ -0,0 +1,213 @@
+{1 Considerations}
+
+When using opam-monorepo the question arises: how to manage it in comparison to
+the regular opam workflow? Compared to regular opam usage using opam-monorepo
+brings lockfiles and a duniverse folder into the repository. There are multiple
+ways to handle this, with multiple advantages and disadvantages. This document
+explains the considerations behind dealing with each of those, along with our
+recommendations.
+
+The document mentions Git for source control as it currently is the most
+popular choice, but similar considerations are valid for most other version
+control systems as well.
+
+{1 Methods}
+
+We assume the opam file is committed to the repository. Running [opam monorepo
+lock] will create a [.locked] opam file in addition to the original opam file.
+Calling [opam monorepo pull] will create a [duniverse] folder into which it
+will unpack the sources of the packages that are mentioned in the lockfile.
+
+{2 Not locked}
+
+While this is not a [opam-monorepo] workflow per se, projects that do not use
+[opam-monorepo] can still be used with [opam-monorepo] and can bring some
+advantages to the project.
+
+In this workflow such every contributor is responsible for creating and
+maintianing their own lockfile and duniverse.
+
+Advantages include:
+
+{ul
+{- No opt-in to [opam-monorepo] required, every developer can chose to use it
+   or not}
+{- Easier management of dependencies over dependencies installed in opam
+   switches}
+{- No merge conflicts when multiple collaborators change versions}
+{- Having the dependencies built by Dune makes cross-compilation easier}
+}
+
+If comes however with some caveats:
+
+{ul
+{- Everyone might end up with different packages in their duniverse, thus low
+   reproducability}
+{- Not self-contained}
+}
+
+{3 Adoption}
+
+Using this workflow is very simple
+
+{[
+$ opam monorepo lock
```

Comment:
unfortunately with the current setting this command is most likely to fail as you first need to add the opam-opervlays remote to the current switch (would be nice to simplify this at one point).

Reply:
I've added a note. With the "explicit repo" feature we should be able to simplify this a bit by adding the opam-overlays repository on the opam-monorepo side automatically, without messing with the users switch. There's a discussion in #273.